ci/slt/slt.sh

```diff
@@ -37,5 +37,6 @@ sqllogictest \
 sqllogictest \
     -v \
     test/sqllogictest/*.slt \
```

Comment:
Are the transform SLTs fast enough to run on every PR? If so, can you add to slt-fast too?

Reply:
I think they should be.